account/views.py

```diff
@@ -1,20 +1,20 @@
 from django.shortcuts import redirect, render, reverse
 
-from django.http import HttpResponse
-
 from .forms import LoginForm, RegisterForm
 
 from django.contrib.auth import authenticate, login, logout
 
+from django.contrib import messages
+
 
 def registration(request):
     if request.method == 'POST':
         form = RegisterForm(request.POST)
         if form.is_valid():
             form.save()
-            return render(request, "account/successful_reg.html")
+            messages.success(request, "Registration passed.")
```

Comment:
`return` is not needed here?

Reply:
I thought not. What is the reason?
The function display RegisterForm with positive or negative feedback. If user see success alert - can decide to open the login tab.

Reply:
Or maybe should I display the success message and then redirect to login page?